index.html

```diff
@@ -3944,283 +3945,242 @@ <h4>updated</h4>
-If a resolving party does not use the <code><a>canonicalId</a></code> value
-as its primary ID value for the DID subject and treat all other equivalent
-values as secondary aliases, there might be negative or unexpected issues that
-arise related to user experience. Implementers are strongly advised to observe the
+is defined to be canonical for the <a>DID subject</a> in the scope of the <a>DID
+document</a>. Like <code><a>equivalentId</a></code>,
+<code><a>canonicalId</a></code> represents a full graph merge because the same
+<a>DID document</a> describes both the <code><a>canonicalId</a></code> DID and
+the <code>id</code> property <a>DID</a>.
+              </p>
+              <p>
+If a resolving party does not use the <code><a>canonicalId</a></code> value as
+its primary ID value for the DID subject and treat all other equivalent values
+as secondary aliases, there might be negative or unexpected issues that arise
+related to user experience. Implementers are strongly advised to observe the
 directives related to this metadata property.
-                  </p>
-                </div>
-
-        </section>
+              </p>
+              <p>
+          </dd>
+        </dl>
       </section>
     </section>
 
     <section>
       <h2>DID URL Dereferencing</h2>
       <p>
-The <a>DID URL dereferencing</a> function dereferences a <a>DID URL</a> into
-a <a>resource</a> with contents depending on the <a>DID URL</a>'s components,
-including the <a>DID method</a>, method-specific identifier, path, query,
-and fragment. This process depends on <a>DID resolution</a> of the <a>DID</a>
+The <a>DID URL dereferencing</a> function dereferences a <a>DID URL</a> into a
+<a>resource</a> with contents depending on the <a>DID URL</a>'s components,
+including the <a>DID method</a>, method-specific identifier, path, query, and
+fragment. This process depends on <a>DID resolution</a> of the <a>DID</a>
 contained in the <a>DID URL</a>. <a>DID URL dereferencing</a> might involve
 multiple steps (e.g., when the DID URL being dereferenced includes a fragment),
-and the function is defined to return the final resource after all
-steps are completed.
-The details of how this process is accomplished are outside the scope of this
-specification, but all conformant implementations implement a function
-which has the following abstract form:
+and the function is defined to return the final resource after all steps are
+completed. The details of how this process is accomplished are outside the scope
+of this specification, but all conforming <a>DID resolvers</a> implement
+the following function which has the following abstract form:
       </p>
 
-      <p><code>
-dereference ( did-url, did-url-dereferencing-input-metadata ) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-url-dereferencing-metadata, content-stream, content-metadata )
-      </code></p>
+      <pre title="Abstract functions for DID Dereferencing">
+dereference(didUrl, options) →
```

Comment:
hmmm, this answers my question about... but it feels odd that this abstract function is defined so far away from the other ones... wonder if we can define them all together in one place, and then address them one at a time like this PR is doing.

Reply:
They're all in the same overall section, but resolution and dereferencing are two very distinct operations and so they're defined separately.

Reply:
Agreed with @jricher -- we're going to keep these two sections separate since they do two very different things.

With my Editor hat off -- I continue to assert that we don't need to define this behavior in a normative way -- the whole dereferencing section is weird -- it's the purview of software libraries that we don't need standards for. We don't standardize the software library interface that an XML parser should use to extract a subsection of an XML document (same for HTML documents). I don't know why we continue to insist that we do this for DID Documents.